packages/api/src/command/internal/update-hie-enabled-feature-flags.ts

```diff
@@ -52,9 +54,15 @@ export async function updateCxHieEnabledFFs({
   } else if (epicEnabled == false) {
     disableFeatureFlagForCustomer(featureFlags.cxsWithEpicEnabled, cxId);
   }
+  if (demoAugEnabled == true) {
+    enableFeatureFlagForCustomer(featureFlags.cxsWithDemoAugEnabled, cxId);
+  } else if (demoAugEnabled == false) {
```

Comment:
I think we should not use `==`, unless we're specifically trying to test for "emptyness" (`x == undefined` to check if `x` is undefined or null) - but I'd rather forbid `==` in the code base altogether than having to deal with the weird behavior of `==` ([ref](https://stackoverflow.com/a/359509/2099911)).

I see it was used for `epicEnabled` (and maybe others). I think it was a mistake and we should fix it too, if not on this PR, soon.

Reply:
It's supposed to be `===`. Fixed.